opentelemetry-sdk/tests/metrics/test_aggregation.py

```diff
@@ -0,0 +1,97 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from math import inf
+from unittest import TestCase
+
+from opentelemetry.sdk._metrics.aggregation import (
+    ExplicitBucketHistogramAggregation,
+    LastValueAggregation,
+    NoneAggregation,
+    SumAggregation,
+)
+
+
+class TestNoneAggregation(TestCase):
+    def test_aggregate(self):
+        """
+        `NoneAggregation` drops all measurements.
+        """
+
+        none_aggregation = NoneAggregation()
+
+        none_aggregation.aggregate(1)
+        none_aggregation.aggregate(2)
+        none_aggregation.aggregate(3)
+
+        self.assertIs(none_aggregation.value, None)
+
+
+class TestSumAggregation(TestCase):
+    def test_aggregate(self):
```

Comment:
SumAggregation also allows negative values, right (e.g. for UpDownCounter)? Should we add a test for that as well?

Reply:
Added a test